playbooks/siege.yml

```diff
@@ -9,7 +9,7 @@
       tags:
         - siege
 
-    # - include_role:
-    #     name: siege-run
-    #   tags:
-    #     - siege
+    - include_role:
+        name: siege-run
+      tags:
+        - siege
```

Comment:
It might be helpful to add one more tag to each step, e.g. `siege-install` or `siege-run`. This way, we could run a piece of this playbook without running it all in one go:

`ansible-playbook -K -t siege-run playbooks/siege.yml`

Reply:
👍